build/os-packages/README.md

```diff
@@ -1,13 +1,17 @@
 # Air-Gap Mode OS Package Build
 
-| OS Distributions | Supported Architectures | Supported container runtimes | Build Method |
-|  ----  | ----  | ----  | ----  |
-| [CentOS 7](https://www.centos.org/) | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
-| [RedHat 7](https://www.redhat.com/en) | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
-| [RedHat 8](https://www.redhat.com/en) | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
-| [Kylin V10](https://www.kylinos.cn/) | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
-| [OpenEuler 22.03](https://www.openeuler.org/) | `amd64` / `arm64` | `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
-| [Oracle Linux 9](https://yum.oracle.com/oracle-linux-isos.html) | `amd64` / `arm64` | `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
+| OS Distributions                                                         | Supported Architectures | Supported container runtimes | Build Method |
+|--------------------------------------------------------------------------| ----  | ----  | ----  |
+| [CentOS 7](https://www.centos.org/)                                      | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
+| [RedHat 7](https://www.redhat.com/en)                                    | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
+| [RedHat 8](https://www.redhat.com/en)                                    | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
+| [RedHat 9](https://www.redhat.com/en)                                    | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
+| [Rocky 9](https://rockylinux.org/news/rocky-linux-9-0-ga-release)        | `amd64` / `arm64` | `docker` / `containerd` | [Automatic build on release](https://github.com/kubean-io/kubean/releases) |
```

Comment:
redhat9 / rocky9 currently does not support docker runtime

Reply:
done